web/vim_lib.js

```diff
@@ -26,6 +26,9 @@
 /*
  * $vimjs holds the common states and functions
  * vimjs_* functions are exposed to C
+ * Multibyte test 1 (3 Norwegian characters): "øæå" done
```

Comment:
Ok, I'll remove. I just thought it would be nice to see instantly if multibyte rendering works.